3rdParty/iresearch/core/analysis/text_token_stream.cpp

```diff
@@ -724,7 +724,7 @@ bool make_json_config(
     // explicit_stopwords_set  marks that even empty stopwords list is valid
     rapidjson::Value stopwordsArray(rapidjson::kArrayType);
     if (!options.explicit_stopwords.empty()) {
-      // for simplifying comparison between properties we need deterministic order of stopwords
+      // for simplifying comparsion between properties we need deterministic order of stopwords
```

Comment:
This was actually correct
```suggestion
      // for simplifying comparison between properties we need deterministic order of stopwords
```

Reply:
`If transpositions are included, does that make it a Damerau–Levenshtein distance?`
@Simran-B yes, correct